.github/workflows/check_style.yml

```diff
@@ -15,15 +15,20 @@ jobs:
       fail-fast: false
       matrix:
         container:
-        - 'px4io/px4-dev-simulation-focal:2020-09-14'
+          - "px4io/px4-dev-simulation-focal:2020-09-14"
     container: ${{ matrix.container }}
     steps:
-    - uses: actions/checkout@v1
-    - name: submodule update
-      run: git submodule update --init --recursive
-    - name: Install clang-format
-      run: apt update && apt install -y clang-format-6.0
-    - name: Check Code format
-      working-directory: Tools
-      run: |
-        ./check_code_format.sh ..
+      - uses: actions/checkout@v1
+      - name: submodule update
+        run: git submodule update --init --recursive
+      - name: Install clang-format
+        run: apt update && apt install -y clang-format-6.0
+      - name: Check Code format
+        working-directory: Tools
+        run: |
+          ./check_code_format.sh ..
+  lint:
```

Comment:
Hugh, didn't know you can do that :)

Reply:
Theoretically, it should also be possible to even directly apply formatting during the commit step. Personally, I prefer to see those changes on saving and then have the pipeline only check, if the styles are consistent (as many of them still allow some room for own format decisions).